### ISSUE TITLE
Can't set shortcut for throw-screen-3

### ISSUE THREAD

**Issue by simcop2387** [ianyh/Amethyst#562]
I've got three monitors and since throwing windows is more common for me than using amethyst to move the mouse, I'm trying to swap the two actions (mod1 and mod2).  I can do it for the first two screens by using the GUI preferences but it doesn't display anything for screen 3 (or 4 if available).

I've tried a ~/.amethyst file that looks like this based on the default.amethyst that came with it but It's having no effect on throw-screen-3 and focus-screen-3.

```javascript
{
    "screens": 3,
    "focus-screen-1": {
        "mod": "mod2",
        "key": "w"
    },
    "focus-screen-2": {
        "mod": "mod2",
        "key": "e"
    },
    "focus-screen-3": {
        "mod": "mod2",
        "key": "r"
    },
    "focus-screen-4": {
        "mod": "mod2",
        "key": "q"
    },
    "throw-screen-1": {
        "mod": "mod1",
        "key": "w"
    },
    "throw-screen-2": {
        "mod": "mod1",
        "key": "e"
    },
    "throw-screen-3": {
        "mod": "mod1",
        "key": "r"
    },
    "throw-screen-4": {
        "mod": "mod1",
        "key": "q"
    },
}
```

**Comment by ianyh** [comment]
What version of the app do you have?

**Comment by simcop2387** [comment]
0.11.2 which according to the app is the newest version available (checked with updates)

**Comment by ianyh** [comment]
Does setting `screens` to 4 work?

**Comment by vkz** [comment]
Just ran into the same issue. Sadly,  I have neither setup nor Swift/Ruby ecosystem but if anyone's set for local build, should be an easy fix: 

https://github.com/ianyh/Amethyst/blob/bb4406e895e6e14a3f7064bc18be8b29df75dd32/Amethyst/Events/HotKeyManager.swift#L337

**Comment by ianyh** [comment]
Fixed by #571
